notebooks/china_data_combine.ipynb

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
<p>Could you add a title, author description cell at the top similar to this: https://github.com/nasaharvest/crop-mask/blob/master/notebooks/train.ipynb ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>